graphql-compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/GraphqlFunctionValidator.java

```diff
@@ -83,12 +83,13 @@ private void validateResourceAccessorName(FunctionDefinitionNode functionDefinit
             if (resourceMethodSymbol.getName().isPresent()) {
                 if (!resourceMethodSymbol.getName().get().equals(PluginConstants.RESOURCE_FUNCTION_GET)) {
```

Comment:
* Could use following style instead.

```suggestion
            Optional<String> methodName = resourceMethodSymbol.getName();
            if (methodName.isPresent()) {
                if (!methodName.get().equals(PluginConstants.RESOURCE_FUNCTION_GET)) {
```